### PR TITLE
Add css styles

### PR DESCRIPTION
**Pull request by cbleslie**
Make it a little more chat like. See PR comments.

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -16,15 +16,11 @@
     <script src="/script.js" defer></script>
   </head>
   <body>
-    <header>
-      <h1>HTML-only chat</h1>
-    </header>
-
```
Comment (cbleslie) [inline views/index.html:22]: Getting rid of this, we can style it and put it back if you want.

File: views/index.html [hunk 2]
```diff
@@ -16,15 +16,11 @@
...
-    <main>
-      <iframe src="/messages"></iframe>
-      <form action="/messages" method="post">
-        <input name="message" type="text" />
-        <button type="submit">submit</button>
+    <main class="c-chat">
+      <iframe class="c-chat__view" src="/messages#latest"></iframe>
+      <form class="c-chat__chat-form" action="/messages" method="post" autocomplete="off">
+        <input class="c-chat__message-input" autofocus name="message" type="text" />
```
Comment (cbleslie) [inline views/index.html:22]: Turned autofocus for the chat input on. So we don't have to keep clicking on it after a message.

File: views/index.html [hunk 3]
```diff
@@ -16,15 +16,11 @@
...
-    </header>
-
-    <main>
-      <iframe src="/messages"></iframe>
-      <form action="/messages" method="post">
-        <input name="message" type="text" />
-        <button type="submit">submit</button>
+    <main class="c-chat">
+      <iframe class="c-chat__view" src="/messages#latest"></iframe>
+      <form class="c-chat__chat-form" action="/messages" method="post" autocomplete="off">
```
Comment (cbleslie) [inline views/index.html:21]: Killed autocomplete; I don't think there is much point for such a light implementation remembering recent messages.

File: public/style.css [hunk 4]
```diff
@@ -1,14 +1,85 @@
...
+  margin: 0;
+  padding: 0;
+  background-color: var(--surface);
 }
 
-iframe {
-  height: 15rem;
+/* Components *****************************************************************/
+
+.c-chat {
```
Comment (cbleslie) [inline public/style.css:37]: Using BEM here, with prefixing. `c` means component.

**Comment by cbleslie** [comment]
Future updates should be to figure out how to autoscroll to the latest message on update, that way we can maintain 100+ recent messages in the message view.

Maybe on load we scroll to an anchor at the bottom of the list by requesting `/messages#latest`?
```HTML
<div id="latest"></div>
```
No idea if this works in an iframe.

**Comment by cbleslie** [comment]
Screen Shot 2020-03-22 at 6 00 52 PM

**Comment by christianbundy** [comment]
Yes! This is beautiful, thank you!

**Comment by christianbundy** [comment]
Screenshot_2020-03-22 Chat

File: views/index.html (hunk 2 above)
Comment (christianbundy) [reply views/index.html:22]: Great idea!

File: views/index.html (hunk 3 above)
Comment (christianbundy) [reply views/index.html:21]: Perfect, that's great.

**Comment by christianbundy** [comment]
> Future updates should be to figure out how to autoscroll to the latest message on update, that way we can maintain 100+ recent messages in the message view.

I think that's a great idea. In my original testing I couldn't get my browser to show HTML until the iframe's HTTP request finished -- which never happens. I used text as a workaround, but would really love to use HTML inside the iframe.

**Comment by cbleslie** [comment]
I have a branch locally that has html rendering. I'll try and make a PR soon.

**Pull request merged**